pkg/k8s/identitybackend/identity.go

```diff
@@ -61,50 +59,35 @@ type crdBackend struct {
 func (c *crdBackend) DeleteAllKeys(ctx context.Context) {
 }
 
-func (c *crdBackend) DeleteID(ctx context.Context, id idpool.ID) error {
-	return c.Client.CiliumV2().CiliumIdentities().Delete(ctx, id.String(), metav1.DeleteOptions{})
+// Select Labels that should be added to CRD CiliumIdentity objects
```

Comment:
nit: can you make it super clear that these are the labels in the `metadata.Labels`, and are thus "advisory" and have no bearing on the actual security identity? Otherwise I foresee confusion.